p2p/dht/bootstrap.go

```diff
@@ -61,6 +62,8 @@ BOOTLOOP:
 		}()
 
 		select {
+		case <-ctx.Done():
+			break BOOTLOOP
```

Comment:
breaking the BOOTLOOP signals the caller that the Bootstrap was completed successfully (note that outside the BOOTLOOP it returns nil i.e. no error) - instead you should return a proper error (add one if no such exists)

Reply:
Updated